vp/devel/headsail.repl

```diff
@@ -0,0 +1,277 @@
+    sysbus 0x1FFF03000
+    }
+
+apb_gpio: GPIOPort.CC2538_GPIO @ {
+    sysbus 0xFFF04000;
+    sysbus 0x1FFF04000
+    }
+
+apb_i2c: I2C.STM32F4_I2C @ {
+    sysbus 0xFFF05000;
+    sysbus 0x1FFF05000
+    }
+
+sysbus:
+    init:
+        Tag <0xFF800000 0x0354> "dsp_0"
+        Tag <0xFF820000 0x10000> "dsp_1"
+        Tag <0xFB000200 0x200> "rgmii_adapter_registers_0"
+        Tag <0xFB000400 0x200> "rgmii_adapter_registers_1"
+        Tag <0xFD000200 0x200> "rgmii_adapter_registers_2"
+        Tag <0xFD000400 0x200> "rgmii_adapter_registers_3"
+        Tag <0xFA200000 0xC000> "mii_dma_port_config_registers_0"
+        Tag <0xFA210000 0x8020> "mii_dma_port_config_registers_1"
+        Tag <0xFA220000 0x8020> "mii_dma_port_config_registers_2"
+        Tag <0xFC200000 0xC000> "mii_dma_port_config_registers_3"
+        Tag <0xFC210000 0x8020> "mii_dma_port_config_registers_4"
+        Tag <0xFC220000 0x8020> "mii_dma_port_config_registers_5"
+        Tag <0xFA800000 0x10000> "switch_config_registers_0"
+        Tag <0xFC800000 0xF000> "switch_config_registers_1"
+        Tag <0xFFE00000 0x1000> "hpc_0"
+        Tag <0xFFE50000 0x1000> "hpc_1"
+        Tag <0xFFA00000 0x0240> "dma_0"
+        Tag <0xFFA08000 0x0240> "dma_1"
+        Tag <0xFE000200 0x001C> "mdio_map_0"
+        Tag <0xFE000400 0x001C> "mdio_map_1"
+        Tag <0xFA190000 0x2008> "frtc_0"
+        Tag <0xFA180000 0x2008> "frtc_1"
+        Tag <0xFC190000 0x2008> "frtc_2"
+        Tag <0xFC180000 0x2008> "frtc_3"
+        Tag <0x100000000 0x1000> "hpc debug" // [^1]
+        Tag <0x100052000 0x1000> "hpc l2 cache config" // [^1]
+        Tag <0x100054000 0x1000> "hpc cluster config" // [^1]
+        Tag <0x1FF800000 0x0354> "dsp_0"
+        Tag <0x1FF820000 0x10000> "dsp_1"
+        Tag <0x1FB000200 0x200> "rgmii_adapter_registers_0"
+        Tag <0x1FB000400 0x200> "rgmii_adapter_registers_1"
+        Tag <0x1FD000200 0x200> "rgmii_adapter_registers_2"
+        Tag <0x1FD000400 0x200> "rgmii_adapter_registers_3"
+        Tag <0x1FA200000 0xC000> "mii_dma_port_config_registers_0"
+        Tag <0x1FA210000 0x8020> "mii_dma_port_config_registers_1"
+        Tag <0x1FA220000 0x8020> "mii_dma_port_config_registers_2"
+        Tag <0x1FC200000 0xC000> "mii_dma_port_config_registers_3"
+        Tag <0x1FC210000 0x8020> "mii_dma_port_config_registers_4"
+        Tag <0x1FC220000 0x8020> "mii_dma_port_config_registers_5"
+        Tag <0x1FA800000 0x10000> "switch_config_registers_0"
+        Tag <0x1FC800000 0xF000> "switch_config_registers_1"
+        Tag <0x1FFE00000 0x1000> "hpc_0"
+        Tag <0x1FFE50000 0x1000> "hpc_1"
+        Tag <0x1FFA00000 0x0240> "dma_0"
+        Tag <0x1FFA08000 0x0240> "dma_1"
+        Tag <0x1FE000200 0x001C> "mdio_map_0"
+        Tag <0x1FE000400 0x001C> "mdio_map_1"
+        Tag <0x1FA190000 0x2008> "frtc_0"
+        Tag <0x1FA180000 0x2008> "frtc_1"
+        Tag <0x1FC190000 0x2008> "frtc_2"
+        Tag <0x1FC180000 0x2008> "frtc_3"
```

Comment:
Let's just not worry about these for now unless they actually become a problem (e.g., overlap).

Reply:
Agreed